cmd/cli/devstack/devstack.go

```diff
@@ -83,34 +78,12 @@ func NewCmd() *cobra.Command {
 			return configflags.BindFlags(viper.GetViper(), devstackFlags)
 		},
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			// ensure we either use a temp repo for the devstack, or the repo path provided
-			// by the specific devstack flag. Never use the default bacalhau repo.
-			repoPath := ODs.ConfigurationRepo
-			if repoPath == "" {
-				// We need to clean up the repo when the node shuts down, but we can ONLY
-				// do this because we know it is a temporary directory. Do not delete the
-				// configured repo if `--stack-repo` was specified
-				repoPath, _ = os.MkdirTemp("", "")
-
-				// If we don't set the repo value in config, readers will be given
-				// a different path to the one we've just created. Presumably a default.
-				defer os.RemoveAll(repoPath)
+			// TODO: a hack to force debug logging for devstack
+			//  until I figure out why flags and env vars are not working
+			if err := logger.ConfigureLogging(string(logger.LogModeDefault), "debug"); err != nil {
+				return fmt.Errorf("failed to configure logging: %w", err)
```

Comment:
They aren't working because a config hasn't been initialized, e.g. via  `util.SetupConfig`, or similar.

Reply:
thats what I thought initially, but we do configure the logger in root cmd and I don't see logging flags in devstack command